crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -369,7 +369,7 @@ pub struct SequenceAction {
     pub rollup_id: RollupId,
     pub data: Vec<u8>,
     /// asset to use for fee payment.
-    pub fee_asset_id: asset::Id,
+    pub fee_asset: asset::Denom,
```

Comment:
we should enforce for all actions that if the denom trace is >32 bytes, it must be in `ibc/` prefixed form. ie don't accept traces that are excessively long as it's a DOS vector

Reply:
Do you mean we should enforce this when parsing `string fee_asset -> asset::Denom`? Technically the upper limit for protobuf payloads is 4MB and sequencer just rejects payloads >200KB, but I can add an additional limit on the type of course.